tinc.yaml

```diff
@@ -0,0 +1,3 @@
+dependencies:
+  - name: riak-protobuf
+    path: protobuf
```

Comment:
Yep, I use cabal, it's far easier and faster.
I don't think mafia has the same clever caching machinery as tinc. Is it?